### PR TITLE
Lvnodcdxi 33

### PR DESCRIPTION
**Pull request by GeFeSTTS**
Add switcherSound component which becomes available when we got sound and voice streams together.

### CODE REVIEW

File: src/client/components/switcherSound/switcherSound.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+    return null;
+  }
+
+  changeTracksVolume = () => {
+    const {
+      volumeValueTrack, volumeValueVoice, track, voice,
+    } = this.state;
+    track.volume = 1 - volumeValueTrack;
+    voice.volume = 1 - volumeValueVoice;
```
Comment (tonypntsrn) [inline src/client/components/switcherSound/switcherSound.js]: Please, don't use magical numbers in code, prefer creating variables and reuse them

File: src/client/components/switcherSound/switcherSound.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
+.SwitcherContainer {
+    background: var(--headerBackground-dark-super);
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    width: 100%;
+    max-width: 1024px;
+    margin: 0 auto;
+    margin: auto;
```
Comment (Kindest13) [inline src/client/components/switcherSound/switcherSound.css]: redundant margin

File: src/client/components/switcherSound/switcherSound.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    -webkit-touch-callout: none;
+}
+
+.SwitcherContainer--slider .rangeslider__handle:after {
+    display: none;
+}
```
Comment (Kindest13) [inline src/client/components/switcherSound/switcherSound.css]: empty line

File: src/client/components/switcherSound/switcherSound.js (hunk 1 above)
Comment (GeFeSTTS) [reply src/client/components/switcherSound/switcherSound.js]: done

File: src/client/components/switcherSound/switcherSound.css (hunk 3 above)
Comment (GeFeSTTS) [reply src/client/components/switcherSound/switcherSound.css]: done

File: src/client/components/switcherSound/switcherSound.css (hunk 2 above)
Comment (GeFeSTTS) [reply src/client/components/switcherSound/switcherSound.css]: done

**Review by mr-redFox — DISMISSED** [review]

**Review by StanislavFedyna — DISMISSED** [review]

**Review by Kindest13 — DISMISSED** [review]

**Review by mr-redFox — DISMISSED** [review]

File: src/client/components/switcherSound/switcherSound.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+class SwitcherSound extends Component {
+  state = {
+    volumeValueTrack: 0.5,
+    volumeValueVoice: 0.5,
+    track: null,
+    voice: null,
+  };
+
+  static getDerivedStateFromProps(props, state) {
+    const { audioData } = props;
```
Comment (tonypntsrn) [inline src/client/components/switcherSound/switcherSound.js]: You could also add here more destructuring 
‘ const { audioData: sound, etc } = props

Comment (GeFeSTTS) [reply src/client/components/switcherSound/switcherSound.js]: done

**Review by Kindest13 — APPROVED** [review]

**Review by mr-redFox — APPROVED** [review]

**Review by tonypntsrn — APPROVED** [review]

**Pull request merged**